src/race.rs

```diff
@@ -50,6 +50,30 @@ impl OnceNonZeroUsize {
         NonZeroUsize::new(val)
     }
 
+    /// Get the reference to the underlying value, without checking if the cell
+    /// is initialized.
+    ///
+    /// # Safety
+    ///
+    /// Caller must ensure that the cell is in initialized state, and that
+    /// the contents are acquired by (synchronized to) this thread.
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    pub fn get_unchecked(&self) -> NonZeroUsize {
```

Comment:
```suggestion
    pub unsafe fn get_unchecked(&self) -> NonZeroUsize {
```

The important part!

Reply:
Haha. Yes, I also need to add some tests and improve the documentation.